scripts/main.js

```diff
@@ -0,0 +1,164 @@
+// main.js
+
+const allSquares = elementsByClassName("square");
+
+function main() {
+    allSquares
+        .forEach(addGameClickListeners);
+
+    _playerToPlay("1");
+}
+
+function addGameClickListeners(item, _) {
+    item.addEventListener('click', function() {
+        _playerSelectedSquare(this);
+    });
+}
+
+main();
+
+// Private Properties
+
+let _gameOver = false;
+let _lastPlayer;
+let _playerOneGame = [];
+let _playerTwoGame = [];
+
```

Comment:
Just out of curiosity, what's the purpose of having the _ infront of the variable and functions? Is that a syntax style thing?

Reply:
An _ prefixing the object or the function indicates it is private